dashboard/app/models/user.rb

```diff
@@ -1067,4 +1071,30 @@ def should_see_inline_answer?(script_level)
     (authorized_teacher? && script && !script.professional_learning_course?) ||
       (script_level && UserLevel.find_by(user: self, level: script_level.level).try(:readonly_answers))
   end
+
+  def show_race_interstitial?
+    if teacher?
+      return false
+    end
+    if under_13?
+      return false
+    end
+    if account_age_days < 7
+      return false
+    end
+    if races && !races.empty?
+      return false
```

Comment:
Is this condition necessary? How would `races` be non-empty without having shown the race interstitial?